### PR TITLE
Campaign crud

### PR DESCRIPTION
**Pull request by AshFromAvalon**
- Implement flatpicker.js
- added columns: start_dat & end_date to campaigns

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -5,7 +5,8 @@
   resources :organisations do
     resources :campaigns, only: [ :new, :create ]
   end
-
+  resources :materials do
+    resources :instructions, only: [:create, :new, :edit, :destroy, :update]
```
Comment (Nicode80) [inline config/routes.rb:9]: nested route only for new and create like campaigns no? No need to have material id for edit, destroy and update as instruction has an id.

File: app/views/organisations/_form.html.erb [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <%= simple_form_for @organisation do |f| %>
     <%= f.input :name, label: false, placeholder: "name*", input_html: { class: "col-12 mb-3" } %>
     <%= f.input :address, label: false, placeholder: "address*", input_html: { class: "input col-12 mb-3" } %>
-    <%= f.input :city, label: false, placeholder: "city*", input_html: { class: "input col-12 mb-3" } %>
-    <%= f.input :postcode, label: false, placeholder: "postcode*", input_html: { class: "input col-12 mb-3" } %>
-    <%= f.input :opening_hours, label: false, placeholder: "opening hours* eg. 8am to 6pm", input_html: { class: "input col-12 mb-3" } %>
+    <%= f.input :city, label: false, placeholder: "city*", input_html: { class: "col-12 mb-3" } %>
+    <%= f.input :postcode, label: false, placeholder: "postcode*", input_html: { class: "col-12 mb-3" } %>
+    <%= f.input :opening_hours, label: false, placeholder: "opening hours* eg. 8am to 6pm", input_html: { class: "col-12 mb-3" } %>
```
Comment (Nicode80) [inline app/views/organisations/_form.html.erb]: City an zipcode not any more in the form thanks to autocomplete.
You might have done it after when merging master ?

Comment (Nicode80) [reply app/views/organisations/_form.html.erb]: OK seen in last commit

File: config/routes.rb (hunk 1 above)
Comment (AshFromAvalon) [reply config/routes.rb:9]: @Nicode80, my PR is for campaigns, not materials. i'm not sure why @mdaudiffret did it this way, we'll have to ask him

**Pull request merged**